File: app/src/androidTest/java/com/example/newsapp/database/DatabaseTest.java
package com.example.newsapp.database;

import android.arch.persistence.room.Room;
import android.support.test.InstrumentationRegistry;

import org.junit.After;
import org.junit.Before;

import data.database.AppDatabase;

public class DatabaseTest {

    protected AppDatabase appDatabase;

    @Before
    public void init() {
        appDatabase = Room.inMemoryDatabaseBuilder(
                InstrumentationRegistry.getContext(),
                AppDatabase.class
        ).build();
    }

    @After
    public void close(){
        appDatabase.close();
    }
}

File: app/src/androidTest/java/com/example/newsapp/database/DatabaseTest.kt
package com.example.newsapp.database

import android.arch.persistence.room.Room
import android.support.test.InstrumentationRegistry

import org.junit.After
import org.junit.Before

import com.example.newsapp.data.database.AppDatabase

open class DatabaseTest {

    protected lateinit var appDatabase: AppDatabase

    @Before
    fun init() {
        appDatabase = Room.inMemoryDatabaseBuilder(
                InstrumentationRegistry.getContext(),
                AppDatabase::class.java
        ).build()
    }

    @After
    fun close() {
        appDatabase.close()
    }
}
